requirements.txt

```
@@ -1 +1,2 @@
-ops >= 2.0.0
\ No newline at end of file
+ops >= 2.0.0
```

Minor thing - I think it would be better to have this requirement specified the following way:

```suggestion
ops == 2.*
```

This way we ensure that these charms will stay in the 2.x series of ops and not jump to ops 3.x if it is ever released. Just an added protection should ops 3.x be radically different from ops 2.x :sweat_smile: